tailwind-theme-preset.js

```diff
@@ -0,0 +1,351 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  theme: {
+    aspectRatio: {
+      '3/1': '3 / 1',
+      '2/1': '2 / 1',
+      '16/9': '16 / 9',
+      '5/3': '5 / 3',
+      '4/3': '4 / 3',
+      '5/4': '5 / 4',
+      '1/1': '1 / 1',
+    },
+    borderRadius: {
+      s: '5px',
+      m: '10px',
+      lg: '30px',
+    },
+    borderWidth: {
+      DEFAULT: '1px',
+      none: '0',
+      0: '0',
+      1: '1px',
+      2: '2px',
+      3: '3px',
+      4: '4px',
+      5: '5px',
+      6: '6px',
+      sm: '1px',
+      md: '2px',
+      lg: '3px',
+      xl: '4px',
+      xxl: '5px',
+      '3xl': '6px',
+    },
+    colors: {
+      none: 'none',
+      transparent: 'transparent',
+      current: 'currentColor',
+      navy: {
+        DEFAULT: '#003654',
```

Comment:
For consistency, is better to use lowercase for all color names.

Reply:
I have tried to rename this variable, but in this case, it is a reserved name. If I rename it, we wouldn't be able to use **'text-navy'** or **'bg-navy'** to set the default value for this color.